exercises/practice/perfect-numbers/.docs/instructions.md

```diff
@@ -0,0 +1,24 @@
+# Instructions
+
+Determine if a number is perfect, abundant, or deficient based on
+Nicomachus' (60 - 120 CE) classification scheme for positive integers.
+
+The Greek mathematician [Nicomachus][nicomachus] devised a classification scheme for positive integers, identifying each as belonging uniquely to the categories of **perfect**, **abundant**, or **deficient** based on their [aliquot sum][aliquot-sum].
+The aliquot sum is defined as the sum of the factors of a number not including the number itself.
+For example, the aliquot sum of 15 is (1 + 3 + 5) = 9
```

Comment:
```suggestion
For example, the aliquot sum of 15 is: (1 + 3 + 5) = 9
```

Reply:
My reasoning for the change is that adding a period to the end of the sentence is confusing, potentially, in terms of programming.  A colon to indicate the statement is not done, providing a statement to the right may provide clarity (and avoid the period).

Reply:
This makes sense. As I understand it changes in primary instructions that are non-raku specific will not be reflected upstream in the problem-specifications repo.

If that's the case, then these suggestions are perhaps better handled there so that they can improve the instructions for all tracks.

Reply:
That is true.  Only suggested here as we can overwrite and have a potential improvement before it comes from upstream.  I still approved it so either way it should be good.